shared/checks/oval/installed_app_is_ocp3.xml

```diff
@@ -0,0 +1,57 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_app_is_ocp3" version="1">
+    <metadata>
+      <title>Red Hat OpenShift Container Platform</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <reference ref_id="cpe:/a:redhat:openshift_container_platform:3.10"
+      source="CPE" />
+      <reference ref_id="cpe:/a:redhat:openshift_container_platform:3.11"
+      source="CPE" />
+      <description>The application installed installed on the system is
+      OpenShift 3.</description>
+    </metadata>
+    <criteria>
+      <extend_definition comment="RHEL7 OS installed" definition_ref="installed_OS_is_rhel7" />
+      <criterion comment="OpenShift Node is installed" test_ref="test_ocp3_node" />
+      <criterion comment="OpenShift Hyperkube is installed" test_ref="test_ocp3_hyperkube" />
+      <criterion comment="Atomic OpenShift is installed" test_ref="test_ocp3_atomic" />
+    </criteria>
```

Comment:
Does it mean that all 3 things have to be installed at the same time?

Reply:
@jan-cerny yes